added SQL database

mellondlancs opened the pull request.

tr4nquil committed:
added SQL database

AAKhan4 reviewed (changes requested):
Conflicting changes to Docker set up, read up docker and docker-compose.yml file. docker-compose service mysql creates database automatically and stuff, issue occurs in newly written code conflicting directly with set up of mysql service. mysql service error shown in unit testing. 

Should sort error by removing CREATE DATABASE command from init.sql and changing appropriate values for name of database being used in docker-compose.yml. Please also bring back previous dummy accounts jhondoe and janedoe with same details for all fields as they are also necessary for unit testing. Ask help if needed, if it does not seem easy to resolve, i will join.

tr4nquil committed:
stopped initialisation of DB and added back test values

AAKhan4 committed:
Merge branch 'main' into 12-setup-mysql

AAKhan4 committed:
Update server.py

change database name for mysql connector

AAKhan4 committed:
Update init.sql

updated database name and edits to comply with docker and previously written backend python

AAKhan4 committed:
Update docker-compose.yml

update database name

AAKhan4 committed:
Update init.sql

KarrimorRifle reviewed (approved):
lgtm, shouldn't have any issues, be sure to test b4 merge
